.github/workflows/purge.yml

```diff
@@ -0,0 +1,24 @@
+name: Purge Cache (jsDelivr)
+
+on:
+  push:
+    paths:
+      - 'packages/synapse-interface/public/pauses/v1/**'
+
+jobs:
+  purge-cache:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Purge CDN Caches
+        uses: gacts/purge-jsdelivr-cache@v1
+        with:
+          url: |
+            https://cdn.jsdelivr.net/gh/synapsecns/sanguine@master/packages/synapse-interface/public/pauses/v1/paused-chains.json
+            https://cdn.jsdelivr.net/gh/synapsecns/sanguine@master/packages/synapse-interface/public/pauses/v1/paused-bridge-modules.json
+            https://cdn.jsdelivr.net/gh/synapsecns/sanguine@test/maintenance/packages/synapse-interface/public/pauses/v1/paused-chains.json
```

Comment:
should nuke these before merge

Reply:
removed test cdn links: https://github.com/synapsecns/sanguine/pull/2857/commits/ee521e1c39219ac5b072e6a31ad6d7f58a3ec456